src/openms/include/OpenMS/ANALYSIS/QUANTITATION/AbsoluteQuantitation.h

```diff
@@ -0,0 +1,250 @@
+      const std::vector<AbsoluteQuantitationStandards::featureConcentration> & component_concentrations,
+      const String & feature_name,
+      const String & transformation_model,
+      const Param & transformation_model_params,
+      std::vector<double> & biases,
+      double & r2_value);
+      
+    /**
+      @brief This function optimizes the parameters of the calibration for a 
+        given component.
+
+      @param component_concentrations list of structures with features and concentrations
+      @param feature_name name of the feature to calculate the absolute concentration.
+      @param transformation_model model used to fit the calibration points
+      @param transformation_model_params parameters used by the transformation_model
+      @param optimized_params optimized parameters
+
+      @exception Exception::UnableToFit
+    */ 
+    void optimizeCalibrationCurveBruteForce(
+      const std::vector<AbsoluteQuantitationStandards::featureConcentration> & component_concentrations,
+      const String & feature_name,
+      const String & transformation_model,
+      const Param & transformation_model_params,
+      Param & optimized_params);
+        
+    /**
+      @brief This function optimizes the parameters of the calibration for a 
+        all components.
+
+    */ 
+    void optimizeCalibrationCurves();    
+
+    /**
+      @brief This function applies the calibration curve to the component.
+
+      @param component the component to be quantified
+      @param IS_component the internal standard (IS) of the component to be quantified.
+        This can be an empty feature if there is no IS for the component.
+      @param feature_name name of the feature to calculate the absolute concentration.
+      @param transformation_model model used to fit the calibration points
+      @param transformation_model_params parameters used by the transformation_model
+
+      @return The absolute concentration.
+
+      @exception Exception::UnableToFit
+    */ 
+    double applyCalibration(const Feature & component,
+      const Feature & IS_component,
+      const String & feature_name,
+      const String & transformation_model,
+      const Param & transformation_model_params);     
+      
+    /**
+      @brief This function applies the calibration curve to all components.
+
+      An additional annotation for metaValue of "calculated_concentration" and "concentration_units"
+        corresponding to the absolute concentration as back-calculated from the
+        fitted calibration curve model and parameters will be added to each sub-feature. 
+        It is assumed that all duplicate components have been removed.  If not,
+        the function will quantify all components, but the first internal standard found will
+        be used to calculate the ratio for the calculation.
+
+      @param unknowns A FeatureMap to quantify.
+
+    */ 
+    void quantifyComponents(FeatureMap& unknowns);    
+
+    void findIS_();
```

Comment:
can you make some clarification about this interface, are all these methods public or are some of them internal? why are only some of these methods in the python bindings?

Reply:
I moved findIS_ to private.  It is an internal class that is not currently used.